sdk/servicebus/azure-servicebus/azure/servicebus/base_handler.py

```diff
@@ -72,14 +72,17 @@ def _build_handler(self):
             encoding=self.encoding,
             **self.handler_kwargs)
 
-    def _mgmt_request_response(self, operation, message, callback, **kwargs):
+    def _mgmt_request_response(self, operation, message, callback, keep_alive_associated_link=True, **kwargs):
```

Comment:
the default value of this new parameter isn't consistent with the one in async_base_handler.py which `False`.

Reply:
and.. where can the user set this parameter?

Reply:
Very good catch on the first part;  I had initially added it as false, seen how many (working) places I'd have to set it to true manually, and changed the default, but forgot to change it here as well.

The user should never set this parameter. (unless they're crafting custom mgmt. requests, which I really assume they're not/shouldn't be) This is really only relevant in terms of the management call specs that we have pre-defined, where most can accept the associated link name property, but a select few can't, so we disable it within that flow internal to our code.

Let me know if this answered your question or if I was off in left field.

Reply:
understood, looks good to me